c/reverse/reverse.c

```diff
@@ -1,14 +1,4 @@
-// Reverse a string in place.
-void reverse(char*s,int len) {
-  char*t = s+len-1,c;
-  while(s<t) {
-    c = *t;
-    *t = *s;
-    *s = c;
-    s++;
-    t--;
-  }
-}
+#include "duktape.h"
```

Comment:
Is this file missing?

Reply:
ahh...sorry... I didn't realize that it would continue to track master.  Let me revert and move this to a branch.

Reply:
Why do you need to rely on the `duktape` library? Do you get errors with the original C code?

Reply:
Ok...that file should be back to the original now.

Reply:
Duktape is an experiment I'm trying to do to get a javascript parser running inside of a canister. I've moved it to a fork for messing around. It was just easier to use the already set up reverse calls and wasm imports in that example to try to get it to work because I don't really understand the API and it isn't documented anywhere.(I haven't gotten it to work....any help would be awesome.)

Reply:
If you have some tips I'm posting over here:  https://forum.dfinity.org/t/help-with-candid-file-construction/1160/7  I'd also love to get back on the dfinity slack.  I think my previous access was revoked.